src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -184,7 +252,7 @@ const NavHeader = (props) => {
           />
         </View>
       )}
-    </Animated.View>
+    </View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Changing the wrapping container from `Animated.View` to `View` might have implications on any animations that were previously applied to this component. Ensure that this change doesn't break any existing animations.



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
it's intended